vcbuild.bat

```diff
@@ -190,6 +192,8 @@ if defined nosnapshot       set configure_flags=%configure_flags% --without-snap
 if defined nonpm            set configure_flags=%configure_flags% --without-npm
 if defined nocorepack       set configure_flags=%configure_flags% --without-corepack
 if defined ltcg             set configure_flags=%configure_flags% --with-ltcg
+:: If clang-cl build is requested, set it to 17.0, which is the version shipped with VS 2022.
```

Comment:
This will work, although it'd be good to add some kind of detection from the system later on.

Reply:
Sure, I just don't know how to do it 😄

Reply:
No worries, it can be a follow-up PR. For now, it's important that we know it should be done.